src/client/debug/drive-robot.tsx

```diff
@@ -181,6 +234,24 @@ export function DriveRobot(props: DriveRobotProps) {
             <Button icon="stop" onClick={handleStopMove}>
                 Stop
             </Button>
+            <Slider
```

Comment:
We should make it clearer these sliders are for joystick driving only. It might even be worth breaking joystick driving into a different component.

Reply:
The sliders are not only for joystick driving. When using the arrow buttons, the sliders update accordingly. The user can also drag the sliders manually to change the motor power levels.